litestar/response/template.py

```diff
@@ -28,13 +28,15 @@ class Template(Response[bytes]):
 
     __slots__ = (
         "template_name",
+        "template_str",
         "context",
     )
 
     def __init__(
         self,
-        template_name: str,
+        template_name: str | None = None,
```

Comment:
I'm not fully committed to the API being called `template_name`, but I, as a person masquerading as a developer, am bad at naming things

Reply:
sgtm